spec/roles.yml

```diff
@@ -83,10 +93,20 @@ components:
             type: string
             minLength: 1
             example: "user"
+        - name: "graph"
+          in: "query"
+          description: "If included in the query returns a graph view of the role"
+          schema:
+            type: string
+            example: ''
 
         responses:
           "200":
-            $ref: 'openapi.yml#/components/responses/Roles'
+            description: "The response body contains the requested role(s)/member(s)"
+            content:
+              application/json:
+                schema:
+                  type: object
```

Comment:
Not breaking, just a question: notice that this is really similar to the existing schema at `openapi.yml#/components/responses/Roles`. Is there an advantage to describing the response schema here?

Reply:
Yeah, don't know what I was thinking here. Made some adjustments